driver-core/src/main/com/mongodb/internal/session/ServerSessionPool.java

```diff
@@ -155,9 +155,10 @@ private void endClosedSessions(final List<BsonDocument> identifiers) {
                 }
             }).getConnection();
 
+            // TODO: how to inject serverApi here, or should we just not?
```

Comment:
I think this has to be done.  Otherwise a server started with requireApiVersion=true will reject this command